### ISSUE TITLE
Unable to access enableWordTimeOffsets config setting from client

### ISSUE THREAD

**Issue by sshadmand** [GoogleCloudPlatform/google-cloud-node#2591]
#### Environment details

  - OS: Mac Os Sierra
  - Node.js version: 6.11.1
  - npm version: 3.10.10
  - google-speech version: 0.10.2

#### Steps to reproduce

  1. require `google-speech`
  2. Use a config for audio transcription like so

```
    const request = {
      audio: {
        uri: fileUri
      },
      config: {
        encoding: 'FLAC',
        languageCode: 'en-US',
        enableWordTimeOffsets: true
      }
    }
```
You will get the error 

```
info: Execution took 9 ms, user function completed successfully
error: ERROR: Error: .google.cloud.speech.v1.RecognitionConfig#enableWordTimeOffsets is not a field: undefined
    at Error (native)
    at MessagePrototype.set (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2490:35)
    at MessagePrototype.set (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2483:38)
    at Message (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2411:34)
    at Element.ProtoBuf.Reflect.ElementPrototype.verifyValue (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:1925:28)
    at T.ProtoBuf.Reflect.FieldPrototype.verifyValue (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:3499:33)
    at MessagePrototype.set (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2493:59)
    at MessagePrototype.set (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2483:38)
    at Message (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/protobufjs/dist/protobuf.js:2411:34)
    at serialize (/Users/seanshadmand/repos/konch/firebase_functions/functions/node_modules/grpc/src/node/src/protobuf_js_5_common.js:81:23)
```

Although I see the 0.10.2 version does not have `enableWordTimeOffsets` in the docs here https://googlecloudplatform.github.io/google-cloud-node/#/docs/speech/0.10.2/speech/v1/speech_client, `enableWordTimeOffsets` does exist in the docs here https://cloud.google.com/speech/reference/rest/v1/RecognitionConfig.

My question is, how can I get to the `enableWordTimeOffsets` using the client (or injecting in manually)?

**Comment by sshadmand** [comment]
I see it is used in the example here as well: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/8384bee28c857def54f2b0b8772f1a04e9d097d3/speech/recognize.js

**Comment by stephenplusplus** [comment]
Thanks for reporting this.

@lukesneeringer the proto file seems to have advanced without us (https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1/cloud_speech.proto). This is already available in a newer release of `google-proto-files` than `@google-cloud/speech` depends on. So, easy fix: I'll update the dependency and bump `@google-cloud/speech`. The only thing is, the option will still be missing from the docs.

To sum up;

- @stephenplusplus: Update Speech module on npm with the latest google-proto-files
- @lukesneeringer: Send a PR from the dummy docs generator (please 😄)

**Comment by stephenplusplus** [comment]
@sshadmand I just published `@google-cloud/speech@0.10.3` with this feature exposed. Let me know if there are any issues.
